tests/IResearch/ExecutionBlockMock-test.cpp

```diff
@@ -87,8 +87,8 @@ struct IResearchBlockMockSetup {
     arangodb::tests::init(true);
 
     // suppress INFO {authentication} Authentication is turned on (system only), authentication for unix sockets is turned on
-    arangodb::LogTopic::setLogLevel(arangodb::Logger::AUTHENTICATION.name(),
-                                    arangodb::LogLevel::WARN);
+    // suppress WARNING {authentication} --server.jwt-secret is insecure. Use --server.jwt-secret-keyfile instead
+    arangodb::LogTopic::setLogLevel(arangodb::Logger::AUTHENTICATION.name(), arangodb::LogLevel::ERR);
```

Comment:
can this change be reverted now? I think the original error with the superfluous log messages is fixed already.

Reply:
it's safe to ignore all errors from the authentication topic anyway, since there are tests to check invalid permissions
and there are quite a few test files with this change, hence I am not sure if it's worthwhile to revert